### PR TITLE
LibWeb: Fix ability to modify selection outside of input

### PR DESCRIPTION
**Pull request by kalenikaliaksandr**
...using keyboard.

Fixes regression introduced in https://github.com/LadybirdBrowser/ladybird/commit/a8077f79cc65a37f82078a6249f161e469e96c3a

### CODE REVIEW

File: Userland/Libraries/LibWeb/Internals/Internals.h [hunk 1]
```diff
@@ -29,6 +29,8 @@ class Internals final : public Bindings::PlatformObject {
     void send_key(HTML::HTMLElement&, String const&, WebIDL::UnsignedShort modifiers);
     void commit_text();
 
+    void send_key_right_keydown_event(WebIDL::UnsignedShort modifiers);
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/Internals/Internals.h:32]: Does `send_key` not work for this? I added it for non-textual keys like this

File: Userland/Libraries/LibWeb/Page/EventHandler.cpp [hunk 2]
```diff
@@ -1023,6 +1023,22 @@ EventResult EventHandler::handle_keydown(UIEvents::KeyCode key, u32 modifiers, u
             FIRE(input_event(UIEvents::EventNames::input, UIEvents::InputTypes::insertText, m_navigable, code_point));
             return EventResult::Handled;
         }
+    } else if (auto selection = document->get_selection(); selection && !selection->is_collapsed()) {
+        if (modifiers & UIEvents::Mod_Shift) {
```
Comment (trflynn89) [inline Userland/Libraries/LibWeb/Page/EventHandler.cpp:1027]: Huh how do we end up here? Seems weird that we'd have no event target

Comment (kalenikaliaksandr) [reply Userland/Libraries/LibWeb/Page/EventHandler.cpp:1027]: we end up here when selection is located outside of input element. maybe I should rename `target` to `input_event_target` to make it more explicit.

File: Userland/Libraries/LibWeb/Internals/Internals.h (hunk 1 above)
Comment (kalenikaliaksandr) [reply Userland/Libraries/LibWeb/Internals/Internals.h]: indeed, it does. I was a bit confused by this function having `HTML::HTMLElement&` as the first argument.

File: Userland/Libraries/LibWeb/Page/EventHandler.cpp (hunk 2 above)
Comment (trflynn89) [reply Userland/Libraries/LibWeb/Page/EventHandler.cpp:1027]: Ah I see, makes sense - I missed/forgot that we get `target` from `active_input_events_target`

**Review by trflynn89 — APPROVED** [review]

**Pull request merged**